dts/bindings/sensor/zephyr,thermistor-rt-table.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# For more information:
+# https://www.electronics-tutorials.ws/io/thermistors.html
```

Comment:
This is the sort of thing that belongs better in the `description` since that's in the HTML pages in https://docs.zephyrproject.org/latest/build/dts/api/bindings.html